### DISCUSSION TITLE
How to set different alert thresholds for specific machines in a business group

### DISCUSSION THREAD

**Discussion by czxin788** [ccfos/nightingale#1205] [Q&A] [answered]
夜莺的告警规则是应用在每个业务组全局的，但怎么给某个业务组下特定的机器设置不同的告警阈值。
比如有一个业务组下的告警规则是，cpu利用率大于75%进行告警，但我想对这个业务组的某个机器，cpu利用率大于90%再告警，其他都还是75%再告警，请问这种场景怎么设置。

是设置不同的业务组告警规则实现吗。

**Comment by UlricQin** [comment, marked as answer]
给不同的机器打不同的标签，在promql里写过滤规则，比如：

```
cpu_usage{ident=~"host1|host2"} > 75
cpu_usage{ident!~"host1|host2"} > 90
```
